🌐 Update German translation for `docs/de/docs/index.md`

nilslindemann opened the pull request:
Based on #3631.

←← `contributing.md` (#10487)
→ `features.md` (#10284)

[German translation progress](https://github.com/tiangolo/fastapi/discussions/10582)

Commit:
German translation of docs/index.md

* Full translation
* Based on #3631
* In sync with current en/docs/index.md
* Hand-translated (Google translate rarely used)
* Checked for errors using LanguageTool
* Manually checked

tiangolo commented:
📝 Docs preview for commit 5a88f0815d35940fe268bd4b67cbeb852371f2cc at: https://ddb946bd.fastapitiangolo.pages.dev

Commit:
Weniger eindeutschen

Die englischen Worte sind in IT-Kreisen verbreiteter.

Siehe die Diskussion bei #10290

tiangolo commented:
📝 Docs preview for commit e9c5ac892a48a156ae8cefa7fd50d04ec73554db at: https://e0b05c9c.fastapitiangolo.pages.dev

Commit:
"Zwei" statt "2"

tiangolo commented:
📝 Docs preview for commit 0dcbfbf025545cb8e329f8ad351624a3790b5a6d at: https://857df007.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/index.md

tiangolo commented:
📝 Docs preview for commit 637db3eac6a97e502b5ba1ac6c3fc63b65ade9de at: https://1aaa4fd8.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/index.md

tiangolo commented:
📝 Docs preview for commit 77237af2c1ddf2d21e74a15a062b07e5be54da20 at: https://bf9ee3f5.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/index.md

tiangolo commented:
📝 Docs preview for commit ddef0c62b6c52e1fdbfc7d37e574401007e34f5a at: https://b0f08d56.fastapitiangolo.pages.dev

Commit:
Better translation for "designed"

Had it on my tongue all the time.

tiangolo commented:
📝 Docs preview for commit ac933c831ce907797d558764dc073d7e105a3751 at: https://8e91c746.fastapitiangolo.pages.dev

Commit:
"Performance" -> "Performanz"

Changed my opinion. https://de.wikipedia.org/wiki/Rechenleistung uses "Performanz", so I use it too.

tiangolo commented:
📝 Docs preview for commit 473edab25ee5b4c12c77e0c2dcde93ebe74c14a9 at: https://846860ab.fastapitiangolo.pages.dev

Commit:
Translate link pointing to async.md

That document is now translated, see https://github.com/tiangolo/fastapi/pull/10449

Also fix another link, which I overlooked.

tiangolo commented:
📝 Docs preview for commit be4abf8f71022643db74954f1f722e8b6ca8a498 at: https://2ae8cb35.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/index.md

tiangolo commented:
📝 Docs preview for commit 22a269ae452869c4b3a97118f5f0212a598bed90 at: https://c6f3deed.fastapitiangolo.pages.dev

Commit:
"Normal" -> "Standard"

tiangolo commented:
📝 Docs preview for commit a182840246a3b1e309c48f15c37d3d1ee1cd9cf9 at: https://ac7870a2.fastapitiangolo.pages.dev

Commit:
Bring translation in sync with #10442

https://github.com/tiangolo/fastapi/pull/10442

tiangolo commented:
📝 Docs preview for commit 201a1558d12198e1ac8193256753b4eca2c15701 at: https://ae0cb77d.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/index.md

tiangolo commented:
📝 Docs preview for commit 69af9ddc71dfd2765ace1fbc938695fe46c5047f at: https://ced9733f.fastapitiangolo.pages.dev

Commit:
Sync with #10392

tiangolo commented:
📝 Docs preview for commit 4ca241dbe6b919497b937d414f70ac027b80dba2 at: https://3f0d31e3.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/index.md

tiangolo commented:
📝 Docs preview for commit 728894c4994ccd5417296cbcc0bfda37e155afed at: https://29baf21d.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/index.md

Commit:
Use typographic quotes

tiangolo commented:
📝 Docs preview for commit b1eda5f53df345ec77cc13129ce7d1d1e84d7c5a at: https://bf8e911c.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/index.md

tiangolo commented:
📝 Docs preview for commit 51a3984dd6a2fa4205b567ccf9875a5f16f08385 at: https://9f92750b.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/index.md

tiangolo commented:
📝 Docs preview for commit b8732d9eee7c794db5918c29b07a6dd1a00e5ca0 at: https://22dcaad2.fastapitiangolo.pages.dev

Commit:
"Standard " -> "Standard-"

tiangolo commented:
📝 Docs preview for commit 3c7b6132b2064ae8bd8bf263e1965e795dfbbea2 at: https://ba939d29.fastapitiangolo.pages.dev

Commit:
Forgot some "Antwort" -> "Response"

tiangolo commented:
📝 Docs preview for commit 603bec7e30ea21696cf0f5a782b879ff080c0256 at: https://4aa13005.fastapitiangolo.pages.dev

Commit:
Forgot some "Zeichenfolge" -> "String"

tiangolo commented:
📝 Docs preview for commit ebddb8bf498301987d24ca4d93a29b0774c94ee5 at: https://e0c4955b.fastapitiangolo.pages.dev

Commit:
"Python " -> "Python-", etc.

tiangolo commented:
📝 Docs preview for commit 01e028ec331aa0c37008b25a7696faa5f8022bd9 at: https://ae7d86c2.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/index.md

tiangolo commented:
📝 Docs preview for commit a8f8bd7a08d4440312376daaea543a2f1fa1c45c at: https://500ccb89.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/index.md

Commit:
"Web-Framework" -> "Webframework"

tiangolo commented:
📝 Docs preview for commit 162eb69a264b8b5bd597c35ef3af54d37c7f57ca at: https://36233420.fastapitiangolo.pages.dev

Commit:
"-Modell" -> "modell"

Commit:
Remove "-"

tiangolo commented:
📝 Docs preview for commit fc8ea97a7ba39c0e61962d4b9224da4444739ebc at: https://831527e2.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/index.md

tiangolo commented:
📝 Docs preview for commit 80366237dc58c50614d67b5547ac0d9c4d4547d7 at: https://e821adbd.fastapitiangolo.pages.dev

Commit:
Update / fix a few things

This was out of sync anyhow.

Commit:
Merge branch 'tiangolo:master' into de/docs/index.md

tiangolo commented:
📝 Docs preview for commit cd178ce572f2bfee3f04af382d68ac58b62780c9 at: https://358fc55f.fastapitiangolo.pages.dev

tiangolo commented:
Awesome @nilslindemann! :rocket: 

As you are the champion of German translations and you have done so many, and with so much attention to detail, I'm gonna go ahead and merge this. :tada: 

If others see things that could be improved later, we can check that in new PRs. :nerd_face:
